src/app/referenceData/reference-data-details/reference-data-details.component.html

```diff
@@ -253,6 +253,7 @@
               [hideDelete]="true"
               [hideCancel]="false"
             ></mdm-editable-form-buttons>
+            <sup *ngIf="result?.readableByEveryone" class="text-muted" style="margin-left: 4px;"><i class="fas fa-globe-europe globe fa-xs" aria-hidden="true" matTooltip="Publicly Available"></i></sup>
```

Comment:
When I was working on custom branding, I found that there were CSS classes already defined for adding margins/padding to introduce spacing. The selectors followed a pattern like:

* `ml-1` represents `margin-left: 4px`
* `mt-2` represents `margin-top: 8px`
* `pr-3` represents `padding-right: 12px`
* Number increments went up by `4px`
* And so on

The full list of selectors can be found in `src\style\abstracts\_helpers.scss`. You don't need to change this for now, but maybe keep in mind for the future.